packages/office-addin-manifest/test/manifests/manifest.json

```diff
@@ -1,120 +1,82 @@
 {
   "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
-  "id": "df2fa15f-332a-47aa-87c2-e30c20686eee",
-  "manifestVersion": "m365DevPreview",
+  "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
```

Comment:
What's the reason for this id?

Reply:
Can be any value. Just liked to be easy to test

Reply:
As long as it's just the test value.